<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverviewFrame</class>
 <widget class="QFrame" name="OverviewFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fff; color: #262626;</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <property name="lineWidth">
   <number>0</number>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="40,0">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>5</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>600</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>489</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Oswald</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>TRANSACTION HISTORY</string>
      </property>
     </widget>
     <widget class="QTreeView" name="m_recentTransactionsView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>400</width>
        <height>500</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>500</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>600</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px; background-color: #eee; color: #262626;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Total:</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_tickerLabel1">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>80</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">CCX</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Unconfirmed:</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_actualBalanceLabel">
      <property name="geometry">
       <rect>
        <x>123</x>
        <y>80</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">0.00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Oswald</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>YOUR WALLET</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_tickerLabel2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>120</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">CCX</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_totalBalanceLabel">
      <property name="geometry">
       <rect>
        <x>103</x>
        <y>160</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">0.00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="m_pendingBalanceLabel">
      <property name="geometry">
       <rect>
        <x>153</x>
        <y>120</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">0.00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="m_tickerLabel3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>160</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">CCX</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Balance:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>400</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Total:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>360</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Unlocked:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Locked:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>221</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Oswald</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>YOUR DEPOSITS</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_tickerLabel6">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>400</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">CCX</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_tickerLabel5">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>360</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">CCX</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_tickerLabel4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>320</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">CCX</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_totalDepositLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>400</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">0.00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="m_unlockedDepositLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>360</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">0.00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="m_lockedDepositLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>320</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">0.00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="m_depositButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>450</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Oswald</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #262626; color: #e5e5e5; border-radius: 5px; padding: 6px;</string>
      </property>
      <property name="text">
       <string>MANAGE DEPOSITS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_depositButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Oswald</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #262626; color: #e5e5e5; border-radius: 5px; padding: 6px;</string>
      </property>
      <property name="text">
       <string>SEND FUNDS</string>
      </property>
     </widget>
     <zorder>label_6</zorder>
     <zorder>m_tickerLabel1</zorder>
     <zorder>label_4</zorder>
     <zorder>m_actualBalanceLabel</zorder>
     <zorder>label</zorder>
     <zorder>m_tickerLabel2</zorder>
     <zorder>m_totalBalanceLabel</zorder>
     <zorder>m_pendingBalanceLabel</zorder>
     <zorder>m_tickerLabel3</zorder>
     <zorder>label_2</zorder>
     <zorder>label_9</zorder>
     <zorder>label_7</zorder>
     <zorder>label_5</zorder>
     <zorder>label_3</zorder>
     <zorder>m_tickerLabel6</zorder>
     <zorder>m_tickerLabel4</zorder>
     <zorder>m_totalDepositLabel</zorder>
     <zorder>m_unlockedDepositLabel</zorder>
     <zorder>m_lockedDepositLabel</zorder>
     <zorder>m_tickerLabel5</zorder>
     <zorder>m_depositButton</zorder>
     <zorder>m_depositButton_2</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
